components/VotesList/VotesListItem.tsx

```diff
@@ -63,9 +64,12 @@ export function VotesListItem({
   const maxDecimals = getPrecisionForIdentifier(decodedIdentifier);
   const Icon = origin === "UMA" ? UMAIcon : PolymarketIcon;
 
-  function formatTitle(title: string) {
-    if (title.length <= 45) return title;
-    return `${title.substring(0, 45)}...`;
+  function onSelectVote(option: DropdownItemT) {
```

Comment:
If the user chooses to do a custom input, then change the input from the dropdown to a normal text input.